package.json

```diff
@@ -41,11 +42,13 @@
   ],
   "author": "Aragon Association",
   "dependencies": {
-    "@radix-ui/react-progress": "^1.0.3",
-    "classnames": "^2.3.2",
+    "@radix-ui/react-progress": "^1.0.0",
+    "classnames": "^2.3.0",
```

Comment:
I thought that it is probably better to have more "relaxed" versions for the dependencies listed under "dependencies" to avoid duplicate dependencies in projects using this library. But your question is legit as this is not documented, I'll add a ticket to investigate about the dependencies version and document how to update/handle them 👍🏻